internal/venice-common/src/main/java/com/linkedin/venice/pubsub/manager/TopicMetadataFetcher.java

```diff
@@ -243,10 +243,15 @@ public void close() throws IOException {
    * @return true if the topic exists, false otherwise
    */
   boolean containsTopic(PubSubTopic topic) {
-    long startTime = System.nanoTime();
-    boolean containsTopic = pubSubAdminAdapter.containsTopic(topic);
-    stats.recordLatency(CONTAINS_TOPIC, startTime);
-    return containsTopic;
+    try {
+      long startTime = System.nanoTime();
+      boolean containsTopic = pubSubAdminAdapter.containsTopic(topic);
+      stats.recordLatency(CONTAINS_TOPIC, startTime);
+      return containsTopic;
+    } catch (Exception e) {
+      LOGGER.error("Failed to check if topic exists: {}", topic, e);
```

Comment:
looks like we log errors in some places and not in other places.
Can we log the error to help with debugging?

Reply:
why not keep them and make them debug? Just in case, we need to know more about those errors in case of spikes of the metric and we have no way to drill down.

Reply:
Sure, we can change the level to debug. 

As I said earlier, it's not like we don't log errors, we do log it in the methods which call this API.

Reply:
Added debug level logs